manim/mobject/svg/text_mobject.py

```diff
@@ -822,6 +823,22 @@ def gen_chars(self):
                 submobjects_char_index += 1
         return chars
 
+    def gen_words(self):
+        words = VGroup()
+        word = VGroup()
+        submobjects_char_index = 0
+        for char_index in range(self.text.__len__()):
+            if not self.text[char_index] in (" ", "\t", "\n"):
```

Comment:
According to line [777](https://github.com/ManimCommunity/manim/pull/1252/files#diff-1cf583cd8792705997f6ee46cd7118aa3d3d52513aeeaab03805586982f02b25R777) `self.text` does not have any of these symbols. Am I missing somethig ? How does it work ?

Reply:
self.gen_words() is called on line 776, just before that happens.